lib/ghtorrent/migrations/030_add_etags.rb

```diff
@@ -0,0 +1,18 @@
+require 'sequel'
+require 'ghtorrent/migrations/mysql_defaults'
+
+Sequel.migration do
+  up do
+    puts 'Adding table etags'
+    create_table :etags do
```

Comment:
Does unique create a unique index on the db?  We should probably also have an id as primary key.

Reply:
Unique creates a unique index in the db. I will be changing it to a bson or gist index. Do you see a value of having an **id** column?